doc/sphinx/compiling.rst

```diff
@@ -32,69 +33,267 @@ Installation Prerequisites
 Linux
 -----
 
-* For Ubuntu or Debian users, the following packages should be installed using
+General Notes
+^^^^^^^^^^^^^
+
+* To download the source code, installing ``git`` is highly recommended.
+
+* SCons is only available for Python 2, so building the Python 3 module requires
+  two installations of Python (one of Python 2 and one of Python 3), even if you
+  do not intend to build the Python 2 module.
+
+* The following instructions use the system-installed versions of Python, but
+  alternate installations such as the Anaconda distribution of Python can be
+  used as well.
+
+* Cython is only required to be installed for the version of Python that also
+  has SCons installed; following the instructions below will install Cython for
+  the version of Python 2 installed in the system directories.
+
+* The SUNDIALS suite of solvers used in Cantera is available from several sources.
+  The sources in most distribution repositories may be out of date, or not
+  optimized for your hardware. The source code for SUNDIALS will be downloaded
+  and compiled with the rest of Cantera, so installing from your distribution's
+  repositories is optional.
+
+* The minimum compatible Cython version is 0.23. If your distribution does not
+  contain a suitable version, you may be able to install a more recent version
+  using ``pip``.
+
+* Building the Fortran interface also requires ``gfortran`` or another supported
+  Fortran compiler.
+
+* Users of other distributions should install the equivalent packages, which
+  may have slightly different names.
+
+.. _sec-ubuntu-debian-reqs:
+
+Ubuntu & Debian
+^^^^^^^^^^^^^^^
+
+* Ubuntu 12.04 LTS (Precise Pangolin) or newer is required; 16.04 LTS (Xenial Xerus)
+  or newer is recommended
+
+* Debian 7.0 (Wheezy) or newer; 8.0 (Jessie) or newer is recommended
+
+* The following packages must be installed to build any of the Cantera modules using
   your choice of package manager::
 
-      g++ python scons libsundials-serial-dev libboost-dev
+      g++ python scons libboost-dev
```

Comment:
Given that SUNDIALS is packaged for Ubuntu/Debian, I don't see a reason not to install it here and use that copy.